.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        ".",
+        "-p",
+        "test_*.py"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": true,
+    "python.testing.cwd": "/Users/kevin/Documents/Development/retworkx/tests"
+}
```

Comment:
This for sure shouldn't be in the PR, maybe we should add `.vscode/settings.json` to `.gitignore`

Reply:
Yikes. We could, though this is usually easy enough to avoid.